### ISSUE TITLE
All menu icons break at once

### ISSUE THREAD

**Issue by oparoz** [nextcloud/server#1047]
### Steps to reproduce

It's going to be difficult to reproduce...
1. Quick-theme your instance
2. Update to latest pre-release
3. Set the language to French
4. Activate some apps

I think this may come from one of the activated app
### Expected behaviour

NC should behave as usual
### Actual behaviour
- All the icons in the menu are broken
- Logo is gone

<img width="217" alt="icons_are_gone" src="https://cloud.githubusercontent.com/assets/323220/17961470/3661e4f2-6aac-11e6-90f8-e4af171e9170.png">

A refresh fixes the issue
### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Updated from an older Nextcloud/ownCloud or fresh install:**

**Where did you install Nextcloud from:**

**Signing status:**
<details>
<summary>

Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

</details>

**List of activated apps:**
<details>
<summary>

App list</summary>

```
Enabled:
  - activity: 2.3.2
  - admin_audit: 1.0.0
  - calendar: 1.3.3
  - comments: 1.0.0
  - dav: 1.0.0
  - federatedfilesharing: 1.0.1
  - federation: 1.0.1
  - files: 1.5.2
  - files_accesscontrol: 1.1.2
  - files_automatedtagging: 1.1.1
  - files_external: 1.0.2
  - files_pdfviewer: 0.8.1
  - files_retention: 1.0.0
  - files_sharing: 1.0.0
  - files_texteditor: 2.1
  - files_trashbin: 1.0.0
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - galleryplus: 15.1.1
  - news: 9.0.2
  - notifications: 0.3.0
  - password_policy: 1.0.0
  - provisioning_api: 1.0.0
  - serverinfo: 1.1.1
  - survey_client: 0.1.5
  - systemtags: 1.0.2
  - tasks: 0.9.3
  - templateeditor: 0.1
  - theming: 1.0.1
  - updatenotification: 1.0.1
  - user_saml: 1.1.0
  - workflowengine: 1.0.1
Disabled:
  - encryption
  - external
  - gallery
  - user_external
  - user_ldap
```

</details>

**The content of config/config.php:**
<details>
<summary>

Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
#### LDAP configuration (delete this part if not used)

<details>
<summary>

LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

</details>
### Client configuration

**Browser:** Chrome

**Operating system:** Windows
#### Web server error log

<details>
<summary>

Web server error log</summary>

```
Insert your webserver log here
```

</details>
#### Nextcloud log (data/nextcloud.log)

<details>
<summary>

Nextcloud log</summary>

```
Insert your Nextcloud log here
```

</details>
#### Browser log

<details>
<summary>

Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by LukasReschke** [comment]
So if one refreshes the issue is gone persistently or only for a moment?

**Comment by LukasReschke** [comment]
Also this is right after the enabling apps step so where the JS injects the new menus. Not on another page, correct?

**Comment by oparoz** [comment]
A refresh fixes it permanently.
It happened only once and it was after app activation. I tried to activate another app and couldn't reproduce.
Unfortunately I wasn't in debug mode, so couldn't see what went wrong. Some call must have been interrupted and the fact that the logo was gone make me think that it's related to theming.

**Comment by oparoz** [comment]
Since the menu is rebuilt while still in the app store, I'm thinking it could have something to do with that script in Files which used to prevent navigation away at some point.

**Comment by MichaelSchmidts** [comment]
I have the same bug, but a refresh dont fix the issue. What can i do?

**Comment by oparoz** [comment]
Try to turn off HTTP/2

**Comment by MichaelSchmidts** [comment]
HTTP/2 is no installed or activated. Only HTTPS

**Comment by oparoz** [comment]
Thanks for the info. Some users have reported issues with HTTP/2. Too bad that wasn't the problem...

**Comment by juliushaertl** [comment]
This could also be related to #2005 

@oparoz @MichaelSchmidts Fix is in #2018, if you'd like to test.

**Comment by MichaelSchmidts** [comment]
I fixed it with the update to 10.0.1. I updated Nextcloud manually because there was also a issue with the automatically update. Nextcloud shows me thats a new update to "Owncloud 9.x.x" is available. So i updated the Nextcloud manually. And after the Update all Icons shows correctly.

**Comment by oparoz** [comment]
I've not experienced the bug since and this was on Chrome, so maybe it was simply a theming issue. Let's close it and I'll re-open if it comes back.
